template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.schemaregistry;
+
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.ConfigUtils.config;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.AbstractRdsRoleConfigProvider;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.template.views.RdsView;
+
+@Component
+public class SchemaRegistryServiceConfigProvider extends AbstractRdsRoleConfigProvider {
```

Comment:
Not a big deal but the class name is a little bit misleading, as no service-level configs are affected here (only DB-related role configs).

Reply:
File Renamed.

Reply:
👍